projects/slap_manipulation/scripts/execution.py

```diff
@@ -0,0 +1,61 @@
+import os
+
+import numpy as np
+import yaml
+
+from home_robot.motion.stretch import STRETCH_HOME_Q, HelloStretchKinematics
+from home_robot_hw.remote import StretchClient
+
+if __name__ == "__main__":
```

Comment:
should probably be def main() with some command line arguments - and should have a comment for how we are expected to use this script?